### PR TITLE
Loadbalancer management network

### PR DESCRIPTION
**Pull request by gthiemonge**

### CODE REVIEW

File: pkg/amphoracontrollers/lb_mgmt_network.go [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+	allPages, err := networks.List(client, listOpts).AllPages()
+	if err != nil {
+		return nil, err
+	}
+	allNetworks, err := networks.ExtractNetworks(allPages)
+	if err != nil {
+		return nil, err
+	}
+	if len(allNetworks) > 0 {
+		return &allNetworks[0], nil
```
Comment (weinimo) [inline pkg/amphoracontrollers/lb_mgmt_network.go:142]: This just returns the first network, not necessarily the lb-mgmt network, right?

Comment (gthiemonge) [reply pkg/amphoracontrollers/lb_mgmt_network.go:142]: it's the first lb-mgmt network, `networks.List` is filtered by the name of the network (and also the tenant of the network)

**Review by beagles — APPROVED** [review]

File: pkg/amphoracontrollers/const.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+	// LbMgmtSubnetGatewayIP -
+	LbMgmtSubnetGatewayIP = ""
+
+	// IPv6 consts
+	// using Unique local address (fc00::/7)
+	// with Global ID 6c:6261:6173 ("lbaas")
+
+	// LbMgmtSubnetIPv6CIDR -
+	LbMgmtSubnetIPv6CIDR = "fd6c:6261:6173:0001::/64"
```
Comment (weinimo) [inline pkg/amphoracontrollers/const.go:52]: I wonder how you came up with this specific CIDR. I assume you could have used any CIDR within fc00::/7, and this one is not particularly easy to remember.

**Review by weinimo — COMMENTED** [review]
LGTM. Just one tiny question about the IPv6 CIDR.

**Review by weinimo — COMMENTED** [review]
/lgtm

**Pull request merged**